Каркас ДевОпс

kasphep opened the pull request:
Зроблений каркас ДевОпс-у українською з посиланнями

kasphep committed:
Каркас ДевОпс

Зроблений каркас ДевОпс-у українською з посиланнями

VolodymyrBalamut commented on docs/devops/orig-tegs.md:
```diff
@@ -0,0 +1,31 @@
+# DevOps
```
Можеш, будь ласка, розповісти для чого використовується цей файл?

kasphep replied:
Оригінальний текст на етапі переводи, поки він не буде ухвалений. Потім видалю, він виключно для моєю та перевіряючого зручності

VolodymyrBalamut reviewed (changes requested):
Треба обговорити питання стосовно того, чи варто додавати цю картинку у наш репозиторій. Чи не буде це порушувати авторські права roadmap.sh.

kasphep committed:
Посилання на джерела

В основних статтях каркасу було змінено шаблон "дна"

kasphep committed:
Правки структури

Підпункти в index.md видалені
Виправлене посилання на статтю до 4 пункту про інтернет
Файл з оригінальними англ тегами видалено
